docs/CHANGELOG.rst

```diff
@@ -13,6 +13,7 @@ Unreleased
 Added
 -----
 * Amplicon workflow
+* Add amplicon descriptor schemas
```

Comment:
Make this consistent with the line above and just use `Amplicon descriptor schemas`